indigo_app/templates/indigo_api/work_popup.html

```diff
@@ -2,39 +2,39 @@
   <h6>
     {{ work.title }}
     {% if work.stub %}
-      <span class="badge badge-info">stub</span>
+    <span class="badge badge-info">stub</span>
```

Comment:
I'd normally indent this because it's under the `if`.

Reply:
Indented